### PR TITLE
feat(stacker): Motor timer bringup

### PR DESCRIPTION
**Pull request by ahiuchingau**
Bringing up the motor timers for XLZ axes with them ticking at 100KHz.

### CODE REVIEW

File: stm32-modules/flex-stacker/firmware/motor_control/motor_policy.cpp [hunk 1]
```diff
@@ -5,11 +5,11 @@
...
+    hw_enable_motor(motor_id);
 }
 
 auto MotorPolicy::disable_motor(MotorID motor_id) -> void {
     disable_motor(motor_id);
 }
 
-auto MotorPolicy::step(MotorID motor_id) -> void { step(motor_id); }
\ No newline at end of file
+auto MotorPolicy::step(MotorID motor_id) -> void { step(motor_id); }
```
Comment (ryanthecoder) [inline stm32-modules/flex-stacker/firmware/motor_control/motor_policy.cpp]:
```suggestion
auto MotorPolicy::step(MotorID motor_id) -> void { step_motor(motor_id); }
```

**Review by ryanthecoder — APPROVED** [review]
Looks good once we fix the linter stuff

File: stm32-modules/flex-stacker/firmware/ui/ui_hardware.c [hunk 2]
```diff
@@ -12,4 +12,7 @@ void ui_hardware_set_heartbeat_led(bool setting) {
     //NOLINTNEXTLINE(performance-no-int-to-ptr)
     HAL_GPIO_WritePin(nSTATUS_LED_GPIO_Port, nSTATUS_LED_Pin,
         setting ? GPIO_PIN_SET : GPIO_PIN_RESET);
+
+//    HAL_GPIO_WritePin(GPIOC, GPIO_PIN_2,
```
Comment (vegano1) [inline stm32-modules/flex-stacker/firmware/ui/ui_hardware.c:16]: unused code

**Review by vegano1 — APPROVED** [review]

**Comment by vegano1** [comment]
Looks good,
Lets get the lint tests to pass and we should be good to go.

**Pull request merged**